src/blocks/recommendations/components/parsely-recommendations-list.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
```

Comment:
I don't think we need the `classNames` package at all. Notice that its raison d'être is to join different classes, but we're not really joining anything.

We could remove it from `package.json` and then have to rerun `npm install` to update `package-lock.json`.